### ISSUE TITLE
Cannot find scikit-geometry on Python online?

### ISSUE THREAD

**Issue by sdg002** [scikit-geometry/scikit-geometry#70]
Hi All,

When I search for `scikit-geometry` on https://pypi.org/ I am unable to find the relevant package.

![image](https://user-images.githubusercontent.com/20245330/129442194-05998bf8-9263-4766-b77b-55687827371b.png)

As a comparison, when I search for `scikit-image` or `scikit-learn` , this is what I get.

![image](https://user-images.githubusercontent.com/20245330/129442237-00840ad6-4b9c-4478-988a-31ecf367fe1c.png)

![image](https://user-images.githubusercontent.com/20245330/129442247-6fd79d64-acb0-49a8-ad41-8d2dffc35534.png)

Any suggestions?

Thanks,
Sau

**Comment by sdg002** [comment]
To provide more context of my problem. 
I was trying to install `scikit-geometry` using `PIP`

`python -m pip install scikit-geometry`
and this threw up this error

```
ERROR: Could not find a version that satisfies the requirement scikit-geometry (from versions: none)
ERROR: No matching distribution found for scikit-geometry
```

**Comment by wolfv** [comment]
Hi, yeah, it's not yet possible to install scikit-geometry from pypi. The package is available through `conda-forge` though. I need to look into how we could create a Python wheel.

**Comment by mjd3** [comment]
[cibuildwheel](https://cibuildwheel.readthedocs.io/en/stable/) could be good here to build wheels for Linux/Mac/Windows for pypi! I have some familiarity with it - could try to put together a PR but probably won't have time until end of September.

**Comment by wolfv** [comment]
@mjd3 that would be awesome. My main question is where we would get CGAL from. Do we need to build it as part of the cibuildwheel process? Or can we rely on some thirdparty repo?

**Comment by mjd3** [comment]
Yeah it could get a little ugly for non-Linux builds for CGAL (for Ubuntu/Debian we can apt-get) but worst case we can always build it from source for those builds. If the goal is to provide a wheel then it doesn't really matter if the wheel building process takes a bit since the install process will be much faster. I can look into this more in detail in a bit!

**Comment by pwuertz** [comment]
You could use [Conan](https://conan.io/) for downloading and managing 3rd-party dependencies in your C++ build. The Conan package repository does indeed provide the latest version of [CGAL](https://conan.io/center/cgal), as well as [Boost](https://conan.io/center/boost) which also appears to be a build requirement.

Using this strategy, I recently wrote a GitHub workflow for building [gdstk](https://github.com/heitzmann/gdstk) PyPI windows packages:

https://github.com/heitzmann/gdstk/blob/master/.github/workflows/pypi-packages.yml

It should be possible to build PyPI packages for `scikit-geometry` this way as well. I've not done this for non CMake based builds yet, but I'm sure Conan has a way for emitting a list of include paths as well, if using CMake is not an option.
